src/ipyniivue_experimental/__init__.py

```diff
@@ -29,9 +30,12 @@ class DRAG_MODES(Enum):
 
 
 def file_serializer(instance: pathlib.Path, widget):
-    if instance == None:
+    if instance is None:
         return None
-    return {"name": instance.name, "data": instance.read_bytes()}
+
+    unique_id = uuid.uuid4()  # This generates a random UUID.
```

Comment:
You'll want to generate an id outside of the serializer. This code will make a unique ID every time, so the front end will never be able to identify the object is the same.

I'd recommend creating a dict or object internally with the field "path", and then using the `id` of that object as the uid for the frontend:

```py
def file_serializer(instance: dict, widget):
    if instance is None:
        return None
    uid = str(id(instance)) # use Python's id to get identifier for our dict (the Image)
    path = instance["path"]
    return { "name": path.name, "data": path.read_bytes(), "uid": uid }
```